CRM/Core/SelectValues.php

```diff
@@ -1109,4 +1109,19 @@ public static function getPermissionedRelationshipOptions() {
     );
   }
 
+  /**
+   * Get option values for dashboard entries (used for 'how many events to display on dashboard').
+   *
+   * @return array
+   *   Dashboard entries options - in practice [-1 => 'Show All', 10 => 10, 20 => 20, ... 100 => 100].
+   */
+  public static function getDashboardEntriesCount() {
+    $optionValues = [];
+    $optionValues[-1] = ts('show all');
```

Comment:
@eileenmcnaughton  why not just empty value here?

Reply:
So given I needed to key by -1 for later handling then I couldn't get it to pass style warnings by just declaring the array - it wanted a space between the minus and the 1

Reply:
(using -1 is a bit dumb but that is what is currently handled when this setting is used)